README.md

```diff
@@ -68,6 +68,54 @@ Prom2teams uses Flask and, to have the service monitored, we use @rycus66's [Pro
 - Use the `--enablemetrics` or `-m` flag when launching prom2teams.
 - Set the environment variable `PROM2TEAMS_PROMETHEUS_METRICS=true`.
 
+### Helm chart
+
+#### Installing the Chart
+
+To install the chart with the release name `my-release` run:
+
+```bash
+$ helm install --name my-release /location/of/prom2teams_ROOT/helm
+```
+
+After a few seconds, Prom2Teams should be running.
```

Comment:
Not very important, but we are using `prom2teams` in lowercase :)

Reply:
:+1: fixed